<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncryptWidget</class>
 <widget class="QWidget" name="EncryptWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>59</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="m_encryptCheckBox">
     <property name="text">
      <string>&amp;Encrypt system</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_passphraseLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Passphrase</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_confirmLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Confirm passphrase</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_iconLabel">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
